composer.json

```diff
@@ -17,7 +17,7 @@
         "psr/log": "^1.0"
     },
     "require-dev": {
-        "openeuropa/code-review": "^0.2",
+        "openeuropa/code-review": "^0.3",
```

Comment:
There is no need to update this to 0.3 if we are not using those features. The client works fine with version 0.2. I suggest to change this to `~0.2` instead, then we are compatible both with 0.2 and 0.3, and even with any future version of the code review tool.